packages/app/src/elements/item-view.ts

```diff
@@ -619,6 +642,44 @@ export class ItemView extends Routing(StateMixin(LitElement)) {
         }
     }
 
+    private async handleDrop(event: DragEvent) {
+        event.preventDefault();
+
+        this._isDraggingFileToAttach = true;
+
+        if (event.dataTransfer?.items) {
+            for (const transferItem of event.dataTransfer.items) {
+                // Only handle files
+                if (transferItem.kind === "file") {
+                    const transferFile = transferItem.getAsFile();
+                    if (transferFile) {
+                        await this._addFileAttachment(transferFile);
+                    }
+                }
+            }
+        } else if (event.dataTransfer?.files) {
+            for (const transferFile of event.dataTransfer.files) {
+                await this._addFileAttachment(transferFile);
+            }
+        }
+
+        this._isDraggingFileToAttach = false;
+    }
+
+    private async handleDragOver(event: DragEvent) {
+        event.preventDefault();
+        this._isDraggingFileToAttach = true;
+    }
+
+    private async handleDragLeave(event: DragEvent) {
+        event.preventDefault();
+        // Only cancel when it's leaving to a different element, since this fires while dragging inside the same element
```

Comment:
This was a tough thing to tweak. I'm not sure if it's because of lit-element or something else, but this fired while dragging, so it kept flickering between the "drop" screen and the item info, without this code.